docs/pages/versions/unversioned/sdk/auth-session.mdx

```diff
@@ -83,6 +83,10 @@ import * as AuthSession from 'expo-auth-session';
 
 <APISection packageName="expo-auth-session" apiName="AuthSession" />
 
+## Providers
+
+AuthSession allows browser-based authentication (using OAuth or OpenID Connect) for Android, iOS, and web. We recommend implementing authentication using native libraries for third-party providers with development builds. For example, use the [`@react-native-google-signin/google-signin`](/guides/google-authentication/) library for Google authentication and [`react-native-fbsdk-next`](/guides/facebook-authentication/) for Facebook. For more information, see [Authentication](/develop/authentication/).
```

Comment:
```suggestion
AuthSession enables general-purpose OAuth and OpenID Connect browser-based auth workflows for Android, iOS, and web. Where available, we recommend using a library supplied by your identity provider, as it will handle implementation details specific to your provider. For example, use [`@react-native-google-signin/google-signin`](/guides/google-authentication/) for Google authentication and [`react-native-fbsdk-next`](/guides/facebook-authentication/) for Facebook. For more information, see [Authentication](/develop/authentication/).
```